tests/e2e/support/objects/app-files/resource/index.ts

```diff
@@ -1,64 +1,7 @@
-import { config } from '../../../../config'
 import { Download, Page } from 'playwright'
-import {
-  createResources,
-  createResourceArgs,
-  deleteResource,
-  deleteResourceArgs,
-  deleteResourceTrashbin,
-  downloadResources,
-  downloadResourcesArgs,
-  downloadResourceVersion,
-  downloadResourceVersionArgs,
-  emptyTrashBinResources,
-  moveOrCopyResource,
-  moveOrCopyResourceArgs,
-  renameResource,
-  renameResourceArgs,
-  restoreResourceVersion,
-  resourceVersionArgs,
-  uploadResource,
-  uploadResourceArgs,
-  dropUploadFiles,
-  restoreResourceTrashbinArgs,
-  restoreResourceTrashbin,
-  searchResourceGlobalSearch,
-  searchResourceGlobalSearchArgs,
-  getDisplayedResourcesFromSearch,
-  clickResource,
-  showHiddenResources,
-  editResources,
-  editResourcesArgs,
-  openFileInViewer,
-  openFileInViewerArgs,
-  expectThatDeleteButtonIsNotVisible,
-  expectThatRestoreResourceButtonVisibility,
-  deleteResourceTrashbinArgs,
-  addTagsToResource,
-  removeTagsFromResource,
-  getTagsForResourceVisibilityInFilesTable,
-  getTagsForResourceVisibilityInDetailsPanel,
-  clickTagArgs,
-  clickResourceTag,
-  getDisplayedResourcesArgs,
-  getDisplayedResourcesFromFilesList,
-  resourceTagsArgs,
-  clickViewModeToggle,
-  expectThatResourcesAreTiles,
-  createSpaceFromFolder,
-  createSpaceFromFolderArgs,
-  createSpaceFromSelection,
-  createSpaceFromSelectionArgs,
-  checkThatFileVersionIsNotAvailable,
-  changePageArgs,
-  changePage,
-  getFileListFooterText,
-  countNumberOfResourcesInThePage,
-  changeItemsPerPage,
-  changeItemsPerPageArgs,
-  expectPageNumberNotToBeVisible
-} from './actions'
+import * as PO from './actions'
```

Comment:
Importing all actions to `PO` (Page Object) so that the import list is reduced to one line. What do you think?

CC @ScharfViktor @SwikritiT @amrita-shrestha @SagarGi @grgprarup @PrajwolAmatya

Reply:
Fine by me))